pkg/i2gw/providers/common/converter.go

```diff
@@ -328,7 +328,9 @@ func (rg *ingressRuleGroup) configureBackendRef(paths []ingressPath) ([]gatewayv
 		backendRefs = append(backendRefs, gatewayv1.HTTPBackendRef{BackendRef: *backendRef})
 	}
 
-	return backendRefs, errors
+	DeDuplicatedBackendRefs := DeDuplicateBackendRefs(backendRefs)
```

Comment:
can we do `uniqueBackendRefs`? I think this name sounds a bit better.  
This comment applies to all of the `DeDuplicatedBackendRefs` mentions in this PR.

Also, consider renaming the function to `removeBackendRefsDuplicates` which is slightly more descriptive.

Reply:
Fixed.